recipes/astropy-base/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = "6.1.3" %}
+
+package:
+  name: astropy-base
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/astropy/astropy-{{ version }}.tar.gz
+  sha256: 9ac834cdedc1f6b5ce6f941f7bfbbfc58fca861eb172bcf72dd90aff8f750970
+
+build:
+  entry_points:
+    - fits2bitmap = astropy.visualization.scripts.fits2bitmap:main
+    - fitscheck = astropy.io.fits.scripts.fitscheck:main
+    - fitsdiff = astropy.io.fits.scripts.fitsdiff:main
+    - fitsheader = astropy.io.fits.scripts.fitsheader:main
+    - fitsinfo = astropy.io.fits.scripts.fitsinfo:main
+    - samp_hub = astropy.samp.hub_script:hub_script
+    - volint = astropy.io.votable.volint:main
+    - wcslint = astropy.wcs.wcslint:main
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<310]
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython >=3.0.0,<3.1.0                          # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - jinja2 >=3
+    - cython >=3.0.0,<3.1.0
+    - setuptools_scm >=6.2
+    - extension-helpers =1
+    - numpy
+  run:
+    - python
+    - numpy >=1.23
+    - pyerfa >=2.0.1.1
+    - astropy-iers-data >=0.2024.7.29.0.32.7
+    - importlib-metadata
+    - pyyaml >=3.13
+    - packaging >=19.0
+
+test:
+  requires:
+    - pytest-astropy
+  commands:
+    - fits2bitmap --help
+    - fitscheck --help
+    - fitsdiff --help
+    - fitsheader --help
+    - fitsinfo --help
+    - samp_hub --help
+    - volint --help
+    - wcslint --help
+  imports:
+    - astropy
+
+about:
+  home: http://www.astropy.org/
+  license: BSD-3-Clause
+  license_file: LICENSE.rst
+  summary: Community-developed Python Library for Astronomy (minimal dependancies variant)
+  description: |
+    The Astropy Project is a community effort to develop a single package for
+    Astronomy in Python. It contains core functionality and common tools
+    needed for performing astronomy and astrophysics research with Python.
+  doc_url: http://docs.astropy.org/en/stable/
+  dev_url: https://github.com/astropy/astropy
+
+extra:
+  recipe-maintainers:
```

Comment:
@astrofrog @bsipocz @mwcraig You all want to maintain this as well as the main astropy-feedstock?

Reply:
an alternative is to use `conda-forge/astropy` which will grandnerd all those users in (but asking is still nice), a la [this examples](https://github.com/conda-forge/dagster-libs-feedstock/blob/main/recipe/meta.yaml#L1838).

It's still good to have _a_ human on the list, so this could be done after the new feedstock is created.

Reply:
sure